test/inductor/test_auto_functionalize.py

```diff
@@ -185,8 +185,7 @@ def f(x, y, z, n):
                     post_grad_graphs,
                     """\
 def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3][1]cpu", arg3_1: "f32[3][1]cpu", arg4_1: "f32[3][1]cpu"):
-        # Custom comment for test
```

Comment:
@yushangdi where is "Custom comment for test" coming from?

Reply:
> @yushangdi where is "Custom comment for test" coming from?

@zou3519 The comment here should be a stack trace. However, the actually stack trace string points to the file location and the file address changes depending on where is your pytorch installed (e.g. /usr/local...). So I just wrote a generic comment. I could write an actual stack trace string, but in this test, call comments are ignored in the comparison.